types/recharts/index.d.ts

```diff
@@ -700,8 +700,8 @@ export class Tooltip extends React.Component<TooltipProps> { }
 
 export interface TreemapProps {
 	data: any[];
-	width: number;
-	height: number;
+	width?: number;
+	height?: number;
```

Comment:
@li-xinyang  Is this somewhere documented?

Reply:
https://github.com/recharts/recharts/blob/master/src/chart/Treemap.js#L193

Basically there is NOT required props for all charts component which is kind pain to use TS with all these props as **required**